Twinpack.iss

```diff
@@ -42,12 +42,13 @@ Source: "TwinpackVsix.17\bin\{#MyConfiguration}\Package\*"; DestDir: "{#TcXaeShe
 Source: "TwinpackCli\bin\{#MyConfiguration}\*"; DestDir: "C:\Program Files (x86)\{#MyAppPublisher}\Twinpack"; Flags: ignoreversion recursesubdirs createallsubdirs; Check: InstallCliInProgramFiles;
 Source: "TwinpackVsix.15\bin\{#MyConfiguration}\TwinpackVsix.15.vsix"; DestDir: "{tmp}"; Flags: deleteafterinstall;
 Source: "TwinpackVsix.17\bin\{#MyConfiguration}\TwinpackVsix.17.vsix"; DestDir: "{tmp}"; Flags: deleteafterinstall;
-Source: "vswhere.exe"; DestDir: "{commonpf}\Zeugwerk\Twinpack"; Flags: ignoreversion;
+Source: "vswhere.exe"; DestDir: "C:\Program Files (x86)\{#MyAppPublisher}\Utils"; Flags: ignoreversion;
```

Comment:
Why do you hardcore C:\Program Files (x86), we have to use Innosetups features here

Reply:
done